Formula/q/qemu.rb

```diff
@@ -62,6 +63,13 @@ class Qemu < Formula
     depends_on "systemd"
   end
 
+  # hvf: arm: disable SME which is not properly handled by QEMU
+  # From https://github.com/utmapp/UTM/blob/v4.6.3/patches/qemu-9.1.2-utm.patch#L714-L41
+  # Needed by macOS 15.2 (or later) on Apple M4 (or later)
+  # https://gitlab.com/qemu-project/qemu/-/issues/2665
+  # https://gitlab.com/qemu-project/qemu/-/issues/2721
+  patch :DATA
```

Comment:
Can we not use the upstream patch?

Reply:
Which patch?

Reply:
I don't think there is an upstream patch.
The issue still happens with `brew install --HEAD` (HEAD-3e9793a):
```console
$ qemu-system-aarch64 -M virt -cpu host -accel hvf
Unexpected error in object_property_find_err() at ../qom/object.c:1345:
qemu-system-aarch64: Property 'host-arm-cpu.sme' not found       
Abort trap: 6 
```

Reply:
Yeap, I haven't found any patch file and so we cannot use a patch url.